src/lib/request.ts

```diff
@@ -1,12 +1,12 @@
 import * as Bluebird from 'bluebird';
-import once = require('lodash/once');
+import * as _ from 'lodash';
```

Comment:
If we're looking for space improvements on the order of kilobytes, `import once from 'lodash/once` results in a smaller size than `import * as _ from 'lodash'`. [Here](https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark) is an interesting article which, while it's talking about bundle size on the frontend, can easily apply to the backend as well.

Reply:
Yeah, I know this is probably more expensive, but the supervisor already imports lodash with `import * as _ from 'lodash'` in most places so changing it here wont have any effect.